### ISSUE TITLE
question about training

### ISSUE THREAD

**Issue by chiehlim** [YunYang1994/tensorflow-yolov3#194]
Hi,

I am still learning about Yolo. Could you help me to understand the training process a bit.

When doing forward pass / prediction using Yolo, the bounding box width and height are calculated by taking the exponent of the raw dwdh from convolution:
pred_wh = (tf.exp(conv_raw_dwdh) * anchors) * stride

When we are doing training, is it correct to say that the loss function find the distance between the pred_wh with the ground truth wh? That means during training, we feed the tensorflow graph with the training image as input, and feed the training object's xy and wh as ground truth. 

Does it make sense to train it this way: convert training image's object wh to raw wh by doing log(wh / (stride * anchors), and then pass this as ground truth the loss function, i.e.loss function finds the distance between conv_raw_dwdh and log(wh / (stride * anchors).

Sorry if this sounds stupid :)
